src/main/java/DianaAssistant.java

```diff
@@ -0,0 +1,155 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class DianaAssistant {
+    private static final String ENCLOSURE = "------------------------------";
+
+    public void interact() {
+        printWelcomeMessage();
+        Scanner scanner = new Scanner(System.in);
+        List<Task> list = new ArrayList<>();
+
+        String input;
+        while (true) {
+            input = scanner.nextLine();
+            if ("bye".equals(input)) {
+                break;
+            }
+
+            switch(getCommand(input)) {
+                case "list":
+                    printList(list);
+                    break;
```

Comment:
The indentation is wrong
```suggestion
            switch(getCommand(input)) {
            case "list":
                printList(list);
                break;
```